server/test/com/cloud/user/AccountManagerImplTest.java

```diff
@@ -185,6 +186,8 @@
     @Mock
     GlobalLoadBalancerRuleDao _gslbRuleDao;
     @Mock
+    SSHKeyPairDao _sshKeyPairDao;
```

Comment:
You've added _sshKeyPairDao here. Without using it! Could you please write a test for the functionality you improved?